Shared/Sources/FocusConfiguration.cpp

```diff
@@ -129,7 +129,8 @@ void FocusConfiguration::generateConfigurationFile(const std::string &configFile
         if (stream) {
             stream << lightconf::config_format::write(config, _source, 80);
         } else {
-            spdlog::get("logger")->error("Unable to create config file");
+            spdlog::get("logger")->critical("Unable to create config file");
```

Comment:
`std::shared_ptr<spdlog::logger>` in constructor [as per the library recommendation](https://github.com/gabime/spdlog/wiki/2.-Creating-loggers#accessing-loggers-using-spdlogget)

>Note: spdlog::get might slow your code down since it locks a mutex, so use with caution. It is advisable to save the shared_ptr<spdlog::logger> returned and use it directly, at least in code hot paths.

Reply:
I'll fix all the spdlog::get in the next Pull request, I'm going to create an issue for that